log-manager/src/main/java/io/airlift/log/Logging.java

```diff
@@ -160,6 +160,15 @@ private static Level getEffectiveLevel(java.util.logging.Logger logger)
         return Level.fromJulLevel(level);
     }
 
+    public void clearLevel(String loggerName)
```

Comment:
is `loggerName` `@Nullable`?

Reply:
It's the same contract as `setLevel`. The `loggers` ConcurrentHashMap already checks for `null` (per the Javadoc: `NullPointerException if the specified key is null`).